coldfront/config/local_settings.py.sample

```diff
@@ -264,6 +264,22 @@ LOCAL_SETTINGS_EXPORT += [
 # ColdFront Plugin Settings
 #===============================================================================
 
+#------------------------------------------------------------------------------
+# Enable django debug toolbar
+#------------------------------------------------------------------------------
+EXTRA_APPS += [
+    'debug_toolbar'
+]
+
+EXTRA_MIDDLEWARE += [
+    'debug_toolbar.middleware.DebugToolbarMiddleware'
+]
+
+INTERNAL_IPS = [
+    '127.0.0.1',
+    '10.0.2.2',
```

Comment:
I'm assuming this is to make the toolbar visible from the Vagrant host.

I can see a use case for having this toolbar on our staging instance (not accessible outside of LBL VPN, so you won't be able to get to it). There, I also had to add my laptop's public IP address to `INTERNAL_IPS` to get the toolbar to show up.

So, I think it'd be useful to add client IP addresses via Ansible. `INTERNAL_IPS = ['127.0.0.1']` could stay here, and then an additional list could be defined in the Ansible-generated `*_settings.py` file. At the bottom of `local_settings.py.sample`, `INTERNAL_IPS` would be extended to include the ones defined in that file.

(One's public IP address can change, so the developer debugging on staging would need to re-run the playbook with their updated address as needed.)